modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -7,7 +7,7 @@ The Try scope allows you to handle errors that may occur when attempting to exec
 
 == Error Handling with the Try Scope
 
-The Try scope has its own error handling strategy that you configure in the same way you configure error handling for a flow. It supports error handling on its inner components instead of the entire flow so that you do not have to extract them into a separate flow.
+The Try scope has its own error handling strategy that you configure in the same way you configure error handling for a flow. The Try scope supports error handling for the components inside the scope instead of the entire flow, so that you do not have to extract them into a separate flow.
```

Comment:
This should not be part of the Mule 4 documentation since it's only relevant for users that come from Mule 3. If we want to put that data somewhere then it should be part of the migration guide documentation,

Reply:
Fixed in 6bc36a1